shared/src/business/entities/WorkItem.ts

```diff
@@ -14,7 +13,7 @@ export class WorkItem extends JoiValidationEntity {
   public completedByUserId?: string;
   public completedMessage?: string;
   public createdAt: string;
-  public docketEntry: any;
```

Comment:
We will remove all uses of `WorkItem` `docketEntry` in app code and drop the column in a followup PR.